packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/DocumentController.java

```diff
@@ -308,6 +318,38 @@ public ResponseEntity<String> getDocumentFileAsText(@PathVariable("id") final St
 
 	}
 
+	/**
+	 * Post Images to Equations Unified service to get an AMR
+	 *
+	 * @param requestMap (Map<String, Object>) JSON request body containing the following fields:
+	 *  	- format		(String) the format of the equations. Options: "latex", "mathml".
+	 *  	- framework (String) the type of AMR to return. Options: "regnet", "petrinet".
+	 *  	- modelId   (String): the id of the model (to update) based on the set of equations
+	 *  	- equations (List<String>): A list of LaTeX strings representing the functions that are used to convert to AMR model
+	 * @return (ExtractionResponse): The response from the extraction service
+	 */
+	@GetMapping("/{id}/image-to-equation")
+	@Secured(Roles.USER)
+	public ResponseEntity<String> postImageToEquation(@PathVariable("id") final String documentId, @RequestParam("filename") final String filename) {
```

Comment:
Can you ping me once this is merged in @blanchco? We'll need to update the logic for this in the TDS migration branch. Thanks!

cc: @kbirk @dgauldie

Reply:
will do!